pages/linux/ufw.md

```diff
@@ -15,9 +15,9 @@
 
 `ufw status numbered`
 
-- Allow incoming traffic on port 5432 on this host:
+- Allow incoming traffic on port 5432 on this host and an identifing comment:
```

Comment:
typo => identifing

And I recommend using "with an" instead of "and an" to signify that this comment is paired with that rule.